wn/web.py

```diff
@@ -320,7 +321,22 @@ async def synset(request):
     return JSONResponse({'data': make_synset(synset, request)})
 
 
+async def index(request: Request):
+    global routes
+    endpoints = [{ 'path': route.path,'name': route.name} for route in routes]
```

Comment:
There are two main problems with this:
1. If you're iterating over `routes`, you'll also get routes that expect path parameters. Let's just include those without any parameters.
2. Just using `route.path` is only getting the relative path. We should use `request.url_for()` to get absolute URLs.

Also, I couldn't find any official way to do a top-level index of endpoints in JSON:API, so what you have is fine, but let's use an object instead of an array:

```suggestion
    endpoints = {
        'lexicons': str(request.url_for('lexicons')),
        'words': str(request.url_for('all_words')),
        'senses': str(request.url_for('all_senses')),
        'synsets': str(request.url_for('all_synsets')),
        'health': str(request.url_for('health_check')),
    }
```